unskript-ctl/unskript-add-check.py

```diff
@@ -68,7 +68,7 @@ def __init__(self):
         except Exception as e:
             print(f"Unable to create __init__.py File {e}")
 
-        AWESOME_DIRECTORY = "Awesome-CloudOps-Automation"
+        AWESOME_DIRECTORY = "awesome"
```

Comment:
This is not the default directory that the Git submodule creates right? are we mentioning this somewhere it needs to be `awesome`?